pyaedt/filtersolutions_core/lumped_topology.py

```diff
@@ -0,0 +1,538 @@
+        return bool(high_low_pass.value)
+
+    @high_low_pass.setter
+    def high_low_pass(self, high_low_pass: bool):
+        status = self._dll.setLumpedHighLowPass(high_low_pass)
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    @property
+    def high_low_pass_min_ind(self) -> bool:
+        """Flag indicating if the high and low pass topology with minimum inductors in synthesized circuit is enabled.
+
+        Returns
+        -------
+        bool
+        """
+        high_low_pass_min_ind = c_bool()
+        status = self._dll.getLumpedHighLowPassMinInd(byref(high_low_pass_min_ind))
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+        return bool(high_low_pass_min_ind.value)
+
+    @high_low_pass_min_ind.setter
+    def high_low_pass_min_ind(self, high_low_pass_min_ind: bool):
+        status = self._dll.setLumpedHighLowPassMinInd(high_low_pass_min_ind)
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    @property
+    def zig_zag(self) -> bool:
+        """Flag indicating if the zig zag topology with minimum inductors in synthesized circuit is enabled.
+
+        Returns
+        -------
+        bool
+        """
+        zig_zag = c_bool()
+        status = self._dll.getLumpedZigZag(byref(zig_zag))
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+        return bool(zig_zag.value)
+
+    @zig_zag.setter
+    def zig_zag(self, zig_zag: bool):
+        status = self._dll.setLumpedZigZag(zig_zag)
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    @property
+    def min_ind(self) -> bool:
+        """Flag indicating if the minimum inductors topology in synthesized circuit is enabled.
+
+        Returns
+        -------
+        bool
+        """
+        min_ind = c_bool()
+        status = self._dll.getLumpedMinInd(byref(min_ind))
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+        return bool(min_ind.value)
+
+    @min_ind.setter
+    def min_ind(self, min_ind: bool):
+        status = self._dll.setLumpedMinInd(min_ind)
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    @property
+    def min_cap(self) -> bool:
+        """Flag indicating if the minimum capacitors topology in synthesized circuit is enabled.
+
+        Returns
+        -------
+        bool
+        """
+        min_cap = c_bool()
+        status = self._dll.getLumpedMinCap(byref(min_cap))
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+        return bool(min_cap.value)
+
+    @min_cap.setter
+    def min_cap(self, min_cap: bool):
+        status = self._dll.setLumpedMinCap(min_cap)
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    @property
+    def set_source_res(self) -> bool:
+        """Flag indicating if the matched source resistor for zig-zag topology in synthesized circuit is enabled.
```

Comment:
Earlier descriptions used zig zag as two words. I prefer the hyphen, but please make the term consistent throughout.